scripts/EdgePy.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
```

Comment:
There is an `__init__.py` in this folder `scripts/`. Why? There should be no module code in `scripts/`.

Reply:
edgepy is the main script - and is thus a module.

Perhaps we're using different definitions of "scripts"

Reply:
Remind me to talk more about putting library code into the modules and submodules and using a script only as a CLI abstraction.